app/views/users/index.html.erb

```diff
@@ -39,12 +23,12 @@
             <td><%= user.address %></td>
             <td>
               <% user.organizations.each do |organization| %>
-                <p><%= organization.name %></p>
+                <span><%= organization.name %></span>
```

Comment:
Why the change to spans? This means it will be hard to correlate organization names with corresponding roles in the next column......